### ISSUE TITLE
Translations & Summon problem

### ISSUE THREAD

**Issue by Light-shield** [SmallCodee/SmallPets#2]
I can't summon pets with translation files. I can only summon them with English. Other languages don't work.

**Comment by SmallCodee** [comment]
Have you also tried the german translations (de)? Because one of the reasons I could think of is that the ' are needed.

**Comment by Light-shield** [comment]
Yes I did that, but it still doesn't work. It only works with English.

**Comment by SmallCodee** [comment]
Is there any error in the console? Is the name of the pet normal, is the message which you get when you reload translated correctly?

**Comment by Light-shield** [comment]
Well all of the plugin is normal except the summon problem. The name of the pets, the messages are normal.

**Comment by SmallCodee** [comment]
I don't really know why.
I tried to recreate the problem, but without luck. For me everything works.
